### PR TITLE
update funding information with 2021/2022 info

### PR DESCRIPTION
**Pull request by hpk42**
also update bitcoin address, and mention on "contribute" page that we are interested in contracting/employing developers

### CODE REVIEW

**Review by r10s — APPROVED** [review]

**Review by Yettte — APPROVED** [review]

**Comment by Yettte** [comment]
Thanks for the update Holger!

**Comment by r10s** [comment]
why was that closed?

probably by accident :) i think, the ui of github is not-so-good here, the close button should be somewhere else :)

**Comment by hpk42** [comment]
maybe @Yettte thought that approving/closing is enough for merging.  But the "Rebase and merge" needs to be clicked explicitely.

File: en/contribute.md [hunk 1]
```diff
@@ -25,19 +25,18 @@ lang: en
...
 - IBAN DE86100777770428658900, Account owner (please always specify): Björn Petersen, BIC NORSDE51XXX
 - PayPal or credit card: [paypal.me/deltachat](https://paypal.me/deltachat/20)
-- Donate regularly via [Liberapay](https://liberapay.com/delta.chat/)
+- Donate via [Liberapay](https://liberapay.com/delta.chat/)
 - Make a one-time or recurring donation via [Open Collective](https://opencollective.com/delta-chat/donate)
-- Send bitcoins to [18e3zwis2raitdZVhEhHHT7xG6oXsZte9L](bitcoin:18e3zwis2raitdZVhEhHHT7xG6oXsZte9L)
-
-We need **devices** for test purposes. Do you have any disused Android Smartphone or iPhone?
-[We](imprint)'d love to take it!
+- Send bitcoins to [bc1qt3xhfvwmdqvxkk089tllvvtzqs8ts06u3u6qka](bitcoin:18e3zwis2raitdZVhEhHHT7xG6oXsZte9L)
```
Comment (dotlambda) [inline en/contribute.md:42]: The link doesn't match the text.

Comment (missytake) [reply en/contribute.md:42]: Erm - thanks for finding this. seems only half of the link was updated. I opened a PR to fix it: https://github.com/deltachat/deltachat-pages/pull/524

**Pull request merged**